zowe/zowe-cli
# Profile API merge error

**What It Does**
Resolves an issue where team config base profiles were being overwritten if a user config did not have a base profile.

**How to Test**
See attached issue for details.

Example Config Files:

zowe.config.user.json:
```
{
    "$schema": "./zowe.schema.json",
    "profiles": {
        "zosmf": {
            "type": "zosmf",
            "properties": {
                "port": <port>
            }
        }
    },
    "defaults": {},
    "autoStore": true
}
```
zowe.config.json:
```
{
    "$schema": "./zowe.schema.json",
    "profiles": {
        "zosmf":{
            "type": "zosmf",
            "properties": {
                "user": <user>,
                "password": <password>
            }
        },
        "global_base": {
            "type": "base",
            "properties": {
                "host": <host>,
                "rejectUnauthorized": false
            }
        }
    },
    "defaults": {
        "zosmf": "zosmf",
        "tso": "tso",
        "ssh": "ssh",
        "base": "global_base"
    },
    "autoStore": false
}
```
**Review Checklist**
I certify that I have:


**Additional Comments**
